app/models/concerns/profile_searching.rb

```diff
@@ -24,6 +24,9 @@ module ProfileSearching
     scope :external, lambda { |external = true|
       where(external: external)
     }
+    scope :internal , lambda {
+      external(false).canonical(false)
```

Comment:
The profile pundit policy already limits to canonical = false - this `canonical(false)` change doesn't mean much if we don't update the policy so users may query for canonical profiles in the first place.

Reply:
This is solved in the backend by https://github.com/RedHatInsights/compliance-backend/pull/497